blog/_posts/2025-07-25-aks-lts-announcement.md

```diff
@@ -0,0 +1,191 @@
+- Reduced testing and validation overhead for workload compatibility
+
+**Enhanced Stability**
+- Production-tested Kubernetes versions with proven track record
+- Minimized risk of introducing breaking changes or regressions
+- Focus on reliability over cutting-edge features
+
+**Compliance and Governance**
+- Extended support timelines that meet enterprise compliance requirements
+- Clear end-of-life planning with advance notice for migration planning
+- Integration with Azure Policy and governance frameworks
+
+## Technical Implementation
+
+**Universal LTS Coverage:** AKS LTS extends to every currently supported Kubernetes version in AKS. Each version undergoes extensive testing across diverse workload patterns and enterprise scenarios to ensure LTS-grade stability and compatibility.
+
+**Beyond Community Support:** When the Kubernetes community support ends (typically 12-14 months after a version's GA), AKS LTS continues providing comprehensive support for 24 months from the original GA date. **Crucially, once the community stops supplying patches after the official version support end-of-life, AKS continues to provide patches and CVE fixes** for all supported components.
+
+This extended support is comprehensive and includes:
+
+- **Core Kubernetes components:** API server, etcd, kubelet, kube-proxy, and all core Kubernetes functionality
+- **AKS-managed add-ons:** (see [**AKS integrations**](https://learn.microsoft.com/en-us/azure/aks/integrations) for the complete list), **including networking, monitoring, and security components**, with the exception of Istio which is coming soon as mentioned in the Looking Ahead section.
+For the current list of unsupported add-ons and features, please refer to our [**LTS unsupported add-ons documentation**](https://learn.microsoft.com/en-us/azure/aks/long-term-support#unsupported-add-ons-and-features)
+- **Node and OS components:** Operating system patches, security updates, and node-level components for both Linux and Windows nodes
+
+This holistic approach ensures that every layer of your full Kubernetes stack remains supported, secure, and stable throughout the entire LTS lifecycle.
+
+**Support Timeline Example:**
+- AKS LTS 1.28 (GA: Nov 2023): Supported until Feb 2026
+- AKS LTS 1.29 (GA: Mar 2024): Supported until Apr 2026
+- AKS LTS 1.30 (GA: Jul 2024): Supported until Jul 2026
+- Future LTS versions will follow the same 24-month support commitment from their respective GA dates of the AKS community versions.
+
+For the most up-to-date information on LTS version support timelines, please refer to the [AKS LTS calendar](https://learn.microsoft.com/en-us/azure/aks/supported-kubernetes-versions?tabs=azure-cli#lts-versions).
+
+
+Throughout the support lifecycle, AKS LTS versions receive:
+- Security patches and vulnerability fixes
+- Fixes for critical bugs that impact stability or functionality
+- Azure service integration updates
+- Compatibility updates for ecosystem tools and applications
```

Comment:
the first two points on security patches and bug fixes were non-ambiguous and clear. these two lines made me as a user mentally question if this is referring to addon/extension/feature minor/functionality updates too. we should probably at some point in the doc establish the addon - AKS minor version pinning policy and that it remains unchanged across community and LTS flavors.